### PR TITLE
fix(bindatafile): overwrites bindatafile if -o flag is set

### PR DESCRIPTION
**Pull request by themihai**

### CODE REVIEW

File: go-bindata-assetfs/main.go [hunk 1]
```diff
@@ -29,6 +29,13 @@ func isDebug(args []string) bool {
 }
 
 func main() {
+	debug := isDebug(os.Args[1:])
```
Comment (elazarl) [inline go-bindata-assetfs/main.go]: Thanks for the PR.
Why won't you put that in a function, like the isDebug parameter?

**Comment by williballenthin** [comment]
Do you have an ETA on this patch being accepted? I've hit this same bug, and would prefer to keep my Go remote URLs pointing at the @elazarl repo.

File: go-bindata-assetfs/main.go (hunk 1 above)
Comment (themihai) [inline go-bindata-assetfs/main.go]: That makes sense but I quickly lost my interest in the repo after finding some other bugs and the lack of any test.

**Comment by pwaller** [comment]
Also hit the same problem.

**Comment by shurcooL** [comment]
In response to a review comment, the author of this Pull Request left a [reply](https://github.com/elazarl/go-bindata-assetfs/pull/17/files#r33417002) that he's not interested in moving this PR forward (he did not close the PR however). So the outlook isn't great.

**Comment by elazarl** [comment]
Since I've seen people are interested with this PR, I'll include it as it is.

I thought that most people already uses different solutions (rice, etc), so I wasn't sure accepting the PR is necessary.

**Pull request merged**